.evergreen/run-tests.sh

```diff
@@ -220,5 +220,9 @@ if [ -z "$GREEN_FRAMEWORK" ]; then
     $PYTHON $COVERAGE_ARGS setup.py $C_EXTENSIONS test $TEST_ARGS $OUTPUT
 else
     # --no_ext has to come before "test" so there is no way to toggle extensions here.
+    # Install the framework from pip.
+    # Use the same flags as the ones used by gevent, in case we have to build the wheel.
+    $PYTHON -m pip install --upgrade pip wheel setuptools
+    $PYTHON -m pip install $GREEN_FRAMEWORK
```

Comment:
Is this using a venv? If not we need to create one first before pip installing anything.

Reply:
It was not, I'll add a note in the new ticket